lib/secretFactory.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const BaseFactory = require('./baseFactory');
+const Secret = require('./secret');
+const iron = require('iron');
+const nodeify = require('./nodeify');
+const hoek = require('hoek');
+// Get symbols for private fields
+const password = Symbol();
+
+let instance;
+
+/**
+ * Seal secret value and return the secret with the sealed value
+ * @method sealSecret
+ * @param  {SecretModel}        secret  Secret to seal
```

Comment:
No. The comment is wrong here because of some refactoring & copy pasta. I should just pass in `string` here.